doc/changelog.rst

```diff
@@ -11,25 +11,34 @@ Jump to :ref:`SmartRedis Changelog <changelog>`
 SmartSim
 ========
 
-Development branch
-------------------
+0.4.2
+-----
 
-To be released at some future date
+Released on April 12, 2023
 
-Note
+Description
 
-This section details changes made in the development branch that have not yet been applied to a released version of the SmartSim library.
+This release of SmartSim had a focus on polishing and extending exiting
+features already provided by SmartSim. Most notably, this release provides
+support to allow users to colocate their models with an orchestrator using
+Unix domain sockets and support for launching models as batch jobs.
 
-Description
+Additionally SmartSim has updated its tool chains to provide a better user
+experience. Most notably, SmarSim can now provide Python 3.10 support. Furthermore,
+by bumping SmartSim now utilizes  SmartRedis's aggregation lists to streamline
```

Comment:
Possibly because of my low English level, the sentence "by bumping SmartSim now utilizes..." sounds a little strange. Plus there is a double space before "SmartRedis". Could you rephrase this?